show/main.py

```diff
@@ -1360,8 +1361,10 @@ def users(verbose):
 @click.option('--silent', is_flag=True, help="Run techsupport in silent mode")
 @click.option('--debug-dump', is_flag=True, help="Collect Debug Dump Output")
 @click.option('--redirect-stderr', '-r', is_flag=True, help="Redirect an intermediate errors to STDERR")
-def techsupport(since, global_timeout, cmd_timeout, verbose, allow_process_stop, silent, debug_dump, redirect_stderr):
+def techsupport(ctx, since, global_timeout, cmd_timeout, verbose, allow_process_stop, silent, debug_dump, redirect_stderr):
     """Gather information for troubleshooting"""
+    if ctx.invoked_subcommand:
+        return
```

Comment:
why return here?

Reply:
We return here to run only the subcommand,  e.g. "show techsupport npu" which can be defined in a vendor platform package, instead of the main "show techsupport".